mrtt-ui/src/views/SiteForm.js

```diff
@@ -81,7 +81,7 @@ const SiteForm = ({ isNewSite }) => {
       site_name,
       landscape_id,
       section_data_visibility: {
-        1: defaultSectionPrivacy,
+        1: 'public',
```

Comment:
Im gunna make this one into a constant because its regression is localized to this component and its easy.

Reply:
sounds good! nice quick work on this one